### ISSUE TITLE
Reduce scoped K43 and Garand recoil by 10%

### ISSUE THREAD

**Issue by x0rnn** [etlegacy/etlegacy#2816]
Since  https://github.com/etlegacy/etlegacy/pull/1680 does not actually make recoil FPS perfectly independent, and it's not possible to limit it exactly to 71 FPS, I closed #2544.

In response, I suggest lowering scoped recoil for K43 and Garand by 10% from the current value, to at least make it a little bit closer to 71 FPS recoil from ETPro/VET.

The arguments as for why are the same as in #2544:

There is a big difference between 100 FPS recoil and 71 FPS, the bigger recoil is game-changing for the experienced sniper and nerfed because of 1 guy who removed the FPS-dependant physics:
>I still get killed in 1s like the recoil doesn't exist while the guy is one pixel on my screen and has no bullet damage drop off. I see no reason to decrease it.

The forced ~100 FPS recoil changes nothing for the average sniper who always used 125 FPS, but worsens the experience for experienced snipers. Making the recoil similar to 71 FPS physics would make both experienced snipers as well as average/beginners happy and improve their sniping as well.

In short, currently it's a major nerf and nobody is happy (except those who hate sniper), making it closer to 71 FPS would make everyone happy (except...) and buff the weapon for noobs a bit.

**Comment by x0rnn** [comment]
K43 recoil etpro:
https://github.com/user-attachments/assets/f31a4d88-1f13-4d12-930e-bdace8b03567

K43 recoil legacy:
https://github.com/user-attachments/assets/e63a13cc-90ac-4182-98c9-7d10deba5e96

etpro:
https://github.com/user-attachments/assets/4879b3e7-b68c-4c03-a25d-670f31eedc3a

legacy:
https://github.com/user-attachments/assets/f9ecf417-10a4-4bf4-b9d9-f33585e6319e

Shooting lower stairs from the same spot in etpro:
![shot0001](https://github.com/user-attachments/assets/17a5f1a0-5801-4ed3-a5b8-01839629c047)

Shooting at the same spot in legacy:
![shot0009](https://github.com/user-attachments/assets/1759d0b3-d73a-4849-9c7e-d54e42816f5d)

**Comment by x0rnn** [comment]
Some more related issues from recent snapshots:

1. In etpro, the black border disappers when you move and shoot - in the above video it doesn't, because in demo playback it's always visible for some reason, but here are 2 videos when actually shooting:

https://github.com/user-attachments/assets/bb3bcd0c-470d-4c2e-b656-74c90c27b2f1

https://discord.com/channels/260750790203932672/346956915814957067/1233444971109945465

In legacy, the black border stays up and scope is locked to the height the recoil hit for a fraction of a second, which is much higher than compared to etpro. It is impossible to efficiently snipe in legacy except if you literally stand still or prone, because the weapon does not allow you to move freely like in the above video, and the higher recoil basically prevents you to reshoot at the same spot no matter your sensitivity or m_pitch. It's just nerfed too much.

**Comment by x0rnn** [comment]
FG42 scope behavior should be separated from K43/Garand if it makes it easier to adjust this.

Also referencing https://github.com/etlegacy/etlegacy/issues/1796 here for consistency.
